spec/index.md

```diff
@@ -1088,6 +1088,29 @@ During these steps, no other entry point of the old or new canister is invoked.
 
 These steps are atomic: If `canister_pre_upgrade` or `canister_post_upgrade` trap, the upgrade has failed, and the canister is reverted to the previous state. Otherwise, the upgrade has succeeded, and the old instance is discarded.
 
+
+:::note 
```

Comment:
I wonder if we need a note here or if we should just make it part of the main text?

Reply:
I am fine with both, @ielashi wdyt?

Reply:
No strong opinion. Either is fine by me.

Reply:
Kept the note, but simplified the surrounding text.